adafruit_display_text/scrolling_label.py

```diff
@@ -110,33 +116,31 @@ def update(self, force=False):
             return
 
     @property
-    def current_index(self):
-        """
-        Index of the first visible character.
+    def current_index(self) -> int:
+        """Index of the first visible character.
 
-        :return int: the current index
+        :return int: The current index
         """
         return self._current_index
 
     @current_index.setter
-    def current_index(self, new_index):
+    def current_index(self, new_index: int):
         if new_index < len(self.full_text):
             self._current_index = new_index
         else:
             self._current_index = new_index % len(self.full_text)
 
     @property
-    def full_text(self):
-        """
-        The full text to be shown. If it's longer than `max_characters` then
+    def full_text(self) -> str:
+        """The full text to be shown. If it's longer than ``max_characters`` then
         scrolling will occur as needed.
 
         :return string: The full text of this label.
```

Comment:
I know this isn't a part of your PR, but if you wouldn't mind changing this, that would be fantastic!

```suggestion
        :return str: The full text of this label.
```